docs/resources/userguide/service/multiclusterservice/mcs-overview.drawio

```diff
@@ -0,0 +1,108 @@
+<mxfile host="Electron" modified="2024-01-09T11:15:29.317Z" agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/21.6.8 Chrome/114.0.5735.289 Electron/25.5.0 Safari/537.36" etag="ZryrKlNv4eWpEWFRcYkw" version="21.6.8" type="device">
```

Comment:
Are these codes the source code of your images? looks good~

Reply:
Yes, it's the source file of draw.io